### PR TITLE
SBERDOMA-962 Extends fields validations from cookies.

### PR DESCRIPTION
**Pull request by MrFoxPro**
This fixes validation error in admin UI when trying to change OrganizationEmployee entity because of lack of fields 'dv', 'sender' in request.

### CODE REVIEW

**Comment by MrFoxPro** [comment]
I suppose this should be refactored

File: apps/condo/domains/common/utils/makeid.utils.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+function makeId (length) {
+    let result = ''
+    const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789'
+    const charactersLength = characters.length
+    for (let i = 0; i < length; i++) {
+        result += characters.charAt(Math.floor(Math.random() * charactersLength))
```
Comment (AntonAL) [inline apps/condo/domains/common/utils/makeid.utils.js:6]: An util `sample` from Lodash can be used here.

**Comment by MrFoxPro** [comment]
@pahaz There are no ways to throw validation error outside of validateInput hook, so I think it is the only way to do this. Or maybe we can extend GQLCustomSchema and inject in every validateInput hook.

File: apps/condo/domains/common/utils/makeid.utils.js (hunk 1 above)
Comment (MrFoxPro) [reply apps/condo/domains/common/utils/makeid.utils.js]: Done

**Review by Dimitreee — APPROVED** [review]

**Review by toplenboren — APPROVED** [review]

**Pull request merged**